applications/MeshMovingApplication/custom_utilities/move_mesh_utilities.cpp

```diff
@@ -78,11 +78,15 @@ ModelPart* GenerateMeshPart(ModelPart &rModelPart,
   const Element &r_reference_element =
       KratosComponents<Element>::Get(rElementName);
 
+
+  // create a new property for the mesh motion
+  Properties::Pointer p_mesh_motion_property = pmesh_model_part->CreateNewProperties(0);
+
   for (int i = 0; i < (int)rModelPart.Elements().size(); i++) {
     ModelPart::ElementsContainerType::iterator it =
         rModelPart.ElementsBegin() + i;
     Element::Pointer p_element = r_reference_element.Create(
-        it->Id(), it->pGetGeometry(), it->pGetProperties());
+        it->Id(), it->pGetGeometry(), p_mesh_motion_property);
```

Comment:
didn't we discuss to revert the changes here?

Reply:
:+1: did it only for the le utilities...

Reply:
Sorry wrong, this is the function that creates the mesh model part for the standard mesh solver, here the property needs to be set.

Reply:
Now as we have the `MESH_POISSON_RATIO` variable, we could keep the property of the element as is and just add the `MESH_POISSON_RATIO` at python level in the solver. Then i would not need to change this function and also the arguments of the strategy would not need to be modified.